src/index.js

```diff
@@ -27,13 +27,17 @@ var Script2 = {
             s.type = 'text/javascript'
             s.appendChild(document.createTextNode(h))
             parent.appendChild(s)
+            this.$emit('load') // any other proper way to do this or emit error?
           })
         } else {
           var opts = _.omitBy(_.pick(this, props), _.isUndefined)
           opts.parent = parent
           // this syntax results in an implicit return
-          var load = () => Script2.load(this.src, opts)
-          _.isUndefined(this.async)
+          var load = () => Script2.load(this.src, opts).then(
+            () => this.$emit('load'),
+            (err) => this.$emit('error', err)
+          )
+          !this.async
```

Comment:
The `_.isUndefined` check was necessary because `<script async ..` means the same thing as `<script async=true`. So we must check for both whether it's undefined, and whether `async="false"` is set, per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script